CPackLists.txt

```diff
@@ -30,6 +30,8 @@
             DESTINATION ${CMAKE_INSTALL_HEADERDIR}/g3log
             COMPONENT headers)
 
+   INSTALL( FILES ${PROJECT_SOURCE_DIR}/cmake/G3LOGConfig.cmake
+            DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/G3LOG)
```

Comment:
Yeah I know that it works. Interesting that the CMake guys give conflicting information in regard to this